docs/release-notes/NuGet-6.10.md

```diff
@@ -191,4 +191,17 @@ Thank you to all the contributors who helped make this NuGet release awesome!
   * [5673](https://github.com/NuGet/NuGet.Client/pull/5673) Enable publishing in VMR
   * [5625](https://github.com/NuGet/NuGet.Client/pull/5625) Eliminate System.CommandLine prebuilt package
 * [brianrob](https://github.com/NuGet/NuGet.Client/pull/5650)
-  * [5650](https://github.com/NuGet/NuGet.Client/pull/5650) Add Restore Instrumentation
\ No newline at end of file
+  * [5650](https://github.com/NuGet/NuGet.Client/pull/5650) Add Restore Instrumentation
+
+## Summary: What's New in 6.10.1
+* [Feature]: add command dotnet config set - [#12476](https://github.com/NuGet/Home/issues/12476)
+
+### Issues fixed in this release
+
+* Promote from warning to error in list package - [#13323](https://github.com/NuGet/Home/issues/13323)
+
+* Visual Studio and PMC restore/update fails when multiple packages.config projects in the solution share the same name (An item with the same key has already been added) - [#13465](https://github.com/NuGet/Home/issues/13465)
+
+* ##[error]The nuget command failed with exit code(1) and error(An item with the same key has already been added. - [#13456](https://github.com/NuGet/Home/issues/13456)
+
+[List of commits in this release](https://github.com/NuGet/NuGet.Client/compare/6.9.1.3...6.10.1.5)
```

Comment:
Given this is under the 6.10.1 heading, should the URL show the differences from 6.10.0, rather than 6.9?